src/poliastro/threebody/cr3bp_char_quant.py

```diff
@@ -0,0 +1,236 @@
+
+    @property
+    def p1(self):
+        """Primary body P1"""
+        return self._p1
+
+    @property
+    def p2(self):
+        """Primary body P2"""
+        return self._p2
+
+    def __mu_calc(self, mu_pi):
+        """Calculate mu of CR3BP
+        Parameters
+        ----------
+        mu_pi : ndarray, float
+            mu_pi[0] = mu of P1
+            mu_pi[1] = mu of P2
+
+        Returns
+        -------
+        mu: float, M2/(M1+M2) = mu2/(mu1+mu2)
+            M1 and M2 are mass of Primary Bodies and M2<M1
+        """
+        return mu_pi[1] / (mu_pi[0] + mu_pi[1])
+
+    def __tstar_calc(self, mu_pi, lstar):
+        """Calculate t* of CR3BP
+        Parameters
+        ----------
+        mu_pi: ndarray, float
+            mu_pi[0] = mu of P1
+            mu_pi[1] = mu of P2
+        lstar: float, km
+            Characterisitc distance between P1 and P2
+
+        Returns
+        -------
+        t*: float, sec
+            Characterisitc time of P1-P2 system
+
+        .. math::
+
+            \sqrt{\frac{l*^3}{M1+M2}}
+        """
+        return (lstar**3 / (mu_pi[0] + mu_pi[1])) ** 0.5
+
+    def BodiesCharCompute(self, p1, p2):
+        """Calculates mu [nd], dist[km], t* [s] of the 'body_i - body_j system'
+        Also, if M2>M1, then swaps p1 and p2, so that M1>M2
+
+        Parameters
+        ----------
+        p1 : string
+           'body_i'
+        p2 : string
+           'body_j'
+
+        Returns
+        -------
+        mu: float, [nd]
+            M2/(M1+M2) = mu2/(mu1+mu2), mu2<mu1
+        lstar: float, km
+            Characterisitc length between P1 and P2
+        tstar: float, sec
+            Characterisitc time of P1-P2 system
+        p1 : string
+           'body_i', more massive body
+        p2 : string
+           'body_j', less massive body
+        """
+
+        mu_pi = []
+        mu_body, distances = self.__bodies_char()
+
+        try:
+            temp1 = mu_body[p1]
+            temp2 = mu_body[p2]
+            if p1 == p2:
+                print(
+                    "Same bodies passed as P1 and P2. Please pass the secondary body of P1 as P2"
+                )
+                return 0, 0, 0, temp1, temp2
+        except KeyError:
+            print(
+                "KeyError-> Incorrect/Bodies Does Not Exist Input Bodies name"
+            )
+            return 0, 0, 0, temp1, temp2
+
+        # Sort P1 and P2 based on their mu values
+        if mu_body[p1] >= mu_body[p2]:  # if p1 is the bigger primary
+            mu_pi.append(mu_body[p1])
+            mu_pi.append(mu_body[p2])
+            bodies = p1 + p2
+        else:  # if p2 is the bigger primary
+            mu_pi.append(mu_body[p2])
+            mu_pi.append(mu_body[p1])
+
+            # Create a string to get distance of the P1-P2 system
+            bodies = p2 + p1
+            # swap p1 and p2, as p1 should be the more massive body
+            temp_pi = p2
+            p2 = p1
+            p1 = temp_pi
```

Comment:
Cool trick to swap variables in Python:
```suggestion
            p2, p1 = p1, p2
```